webui/react/src/pages/F_ExpList/F_ExperimentList.settings.ts

```diff
@@ -114,10 +114,10 @@ export interface F_ExperimentListGlobalSettings {
 
 const ioExpListView = union([literal('scroll'), literal('paged')]);
 
-export const experimentListGlobalSettingsConfig = type({
+export const experimentListGlobalSettingsConfig = {
   expListView: ioExpListView,
   rowHeight: ioRowHeight,
-});
+};
```

Comment:
just to make sure, using `partial` instead of `type` here doesn't work?

Reply:
I had thought that using `partial()` would be necessary to un-do for `userSettings.set`, but it seems to work? That would be a simpler, smaller PR if it checks out logically.

Reply:
ok cut down to 2-line change

Reply:
I think `partial` in both places is actually more correct. The Settings form doesn't actually let you unset these fields but in the application logic it's fine to unset them.

Reply:
ok, rolling back this commit

Reply:
Why?